src/ProgressMeter.jl

```diff
@@ -65,6 +65,12 @@ function BarGlyphs(s::AbstractString)
 end
 const defaultglyphs = BarGlyphs('|','█', Sys.iswindows() ? '█' : ['▏','▎','▍','▌','▋','▊','▉'],' ','|',)
 
+# Internal struct for holding threading information
+Base.@kwdef mutable struct ThreadingInfo
+    detected::Bool = false
+    threads_used::Vector{Bool} = fill(false, Threads.nthreads())
```

Comment:
Threadid may return values bigger than nthreads.

When foreign threads are used (or even just with an interactive thread) iirc

Reply:
Oh, ok. Back to beginning.